detection_rules/docs.py

```diff
@@ -47,16 +49,16 @@ def _get_attack_coverage(self):
         coverage = defaultdict(lambda: defaultdict(lambda: defaultdict(int)))
 
         for rule in self.package.rules:
-            threat = rule.contents.get('threat')
```

Comment:
It looks like there are some other calls to `rule.contents.get()` in this file.

Reply:
i think those are all fixed now 🤞 .
lemme know if i missed some